script.js

```diff
@@ -0,0 +1,338 @@
+    subtractionBtn.style.color = "hsl(52, 100%, 62%)"
+    subtractionBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    decimalPoint.style.background = "hsl(268, 47%, 21%)"
+    decimalPoint.style.color = "hsl(52, 100%, 62%)"
+    decimalPoint.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    divisionBtn.style.background = "hsl(268, 47%, 21%)"
+    divisionBtn.style.color = "hsl(52, 100%, 62%)"
+    divisionBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    multiplicationBtn.style.background = "hsl(268, 47%, 21%)"
+    multiplicationBtn.style.color = "hsl(52, 100%, 62%)"
+    multiplicationBtn.style.boxShadow = "0px 4px 0px hsl(290, 70%, 36%)"
+    allClear.style.background = "hsl(281, 89%, 26%)"
+    allClear.style.color = "white"
+    allClear.style.boxShadow = "0px 4px 0px hsl(285, 91%, 52%)"
+    equalityBtn.style.background = "hsl(176, 100%, 44%)"
+    equalityBtn.style.color = "hsl(198, 20%, 13%)"
+    equalityBtn.style.boxShadow = "0px 4px 0px hsl(177, 92%, 70%)"
+}
+// THE CALCULATOR CODE STARTS HERE
+let clearButton = document.querySelector(".allClear")
+let deleteButton = document.querySelector(".deleteBtn")
+// let calculatorDisplay = document.getElementById("calculatorDisplay");
+let secondDisplayContainer = document.querySelector(".secondDisplayContainer");
+let signs = document.querySelector(".signs");
+// let decimalPoint = document.querySelector(".decimalPoint")
+let NumberButton = document.querySelectorAll(".operands")
+let firstDisplay = document.querySelector(".first-display")
+let dummyDisplay = document.querySelector(".dummy-display")
+let add = document.querySelector(".additionBtn")
+let subtract = document.querySelector(".subtractionBtn")
+let multiply = document.querySelector(".multiplicationBtn")
+let divide = document.querySelector(".divisionBtn")
+let operationButton = document.querySelector(".equalsToBtn")
+let secondDisplay = document.createElement("div");
+NumberButton.forEach(button => {
+   button.addEventListener("click",eachnumber)
+    function eachnumber(){
+      firstDisplay.textContent += button.textContent
+      add.disabled = false
+      subtract.disabled = false
+      multiply.disabled = false
+      divide.disabled = false
+    }
+});
+add.addEventListener("click",added)
+subtract.addEventListener("click",subtracted)
+multiply.addEventListener("click",multiplied)
+divide.addEventListener("click",divided)
+decimalPoint.addEventListener("click", decima)
+operationButton.addEventListener("click",operate)
+clearButton.addEventListener("click",allclear)
+deleteButton.addEventListener("click",deleteCurrentDisplay)
+function decima(){
+    firstDisplay.textContent += "."
+    if(firstDisplay.textContent.includes(".")){
+        decimalPoint.disabled = true
+    }
+    else{
+        decimalPoint.disabled = false
+    }
+}
+function deleteCurrentDisplay(){
+ firstDisplay.textContent = firstDisplay.textContent.substring(0, firstDisplay.textContent.length -1);
+}
```

Comment:
There's a subtle bug here. Say the last character the user enters is a decimal and if they delete it, they are not able to enter a decimal again(because of L205).